src/app/item-popup/ItemTalentGrid.tsx

```diff
@@ -106,6 +106,7 @@ function ItemTalentGrid({ item, perksOnly, bestPerks }: Props) {
                 r="16"
                 cx="-17"
                 cy="17"
+                id={node.hash.toString()}
```

Comment:
```suggestion
```

Reply:
Sorry, testing out the GitHub suggested changes thing. I'd added this to make it easier to figure out the perks for subclasses, but we don't need it anymore.

Reply:
Thanks, I've removed it 👍